lib/scss/6_components/_conversations.scss

```
@@ -1498,3 +1482,36 @@ $color-background-button-modalities: rgba(255, 255, 255, 0.3);
     font-weight: bold;
     font-size: $spacing-unit / 2;
 }
+
+.Progress-status{
+    display: table;
+    margin: 0 auto;
+
+    .Progress-porcentage{
+        font-weight: bold;
+        display: table-cell;
+        font-size: $spacing-unit-tiny * 2;
+        padding-right: $spacing-unit-tiny;
+    }
+    .Progress-absolute{
+        display: table-cell;
+        font-size: $spacing-unit-tiny * 2;
+        padding-left: $spacing-unit-tiny;
+    }
+    .Progress-bar{
+        display: table-cell;
+        width: 80%;
+        &-border{
+            background: transparent;
+            border: 1px solid  $color-primary;
+            border-radius: $spacing-unit-tiny;
+            height: $spacing-unit-tiny;
+        }
+        &-content{
+            height: $spacing-unit-tiny;
+            background: $color-hint;
+            border-top-left-radius: $spacing-unit-tiny;
```

Para as bordas, temos valores definidos nas variáveis no arquivo de spacing...
```suggestion
            border-top-left-radius: $border-radius-tiny;
```